src/commands/index.js

```diff
@@ -3,7 +3,7 @@ const Promise = require('bluebird');
 
 const REGISTRY = require('./registry');
 
-const CMD_FLAG_REGEX = new RegExp(/^-(\w{1})$/);
+const CMD_FLAG_REGEX = new RegExp(/^-(\w{1,5})$/);
```

Comment:
Here is what I suggest - this should allow full word flags and any number of shorthand chained flags.

```suggestion
const CMD_FLAG_REGEX = new RegExp(/^(--?\w+)$/);
```